### PR TITLE
Patch compara test dbs

### PR DESCRIPTION
**Pull request by carlacummins**
### Description

Patch compara test dbs to e101 schema

### Use case

There have been changes to the compara schema that should be reflected here. These changes are required for `release/101` branching!

### CODE REVIEW

**Comment by carlacummins** [comment]
The changed code looks fine to me - I think the rogue commit isn't really making any difference to the code

File: t/test-genome-DBs/homology/compara/table.sql [hunk 1]
```diff
@@ -321,6 +321,7 @@ CREATE TABLE `genomic_align_block` (
   `length` int(10) NOT NULL,
   `group_id` bigint(20) unsigned DEFAULT NULL,
   `level_id` tinyint(2) unsigned NOT NULL DEFAULT '0',
+  `direction` int(1) unsigned DEFAULT NULL,
```
Comment (muffato) [inline t/test-genome-DBs/homology/compara/table.sql]:
```suggestion
  `direction` tinyint(1) unsigned DEFAULT NULL,
```

File: t/test-genome-DBs/multi/compara/table.sql [hunk 2]
```diff
@@ -319,6 +319,7 @@ CREATE TABLE `genomic_align_block` (
   `length` int(10) NOT NULL,
   `group_id` bigint(20) unsigned DEFAULT NULL,
   `level_id` tinyint(2) unsigned NOT NULL DEFAULT '0',
+  `direction` int(1) unsigned DEFAULT NULL,
```
Comment (muffato) [inline t/test-genome-DBs/multi/compara/table.sql]:
```suggestion
  `direction` tinyint(1) unsigned DEFAULT NULL,
```

**Comment by carlacummins** [comment]
Yes, Travis failures look like a variation issue to me

**Pull request merged**